### ISSUE TITLE
Increase default size efi 

### ISSUE THREAD

**Issue by Cardiacman13** [archlinux/archinstall#2124]
Please increase the default EFI size to 1 GB.

With 2 kernels, one Zen and one LTS, plus the Nvidia modules, we are running out of space.

Thx :)

**Comment by Torxed** [comment]
We've recently made it possible to separate `EFI` and `/boot` which has been the biggest issue.
Is that enough, or does the `EFI` still need to be larger?

*(the split should make it so that a ~512MB EFI fits the UEFI binaries, and `/boot` can house the `vmlinuz` and initramfs)*

**Comment by Cardiacman13** [comment]
![image](https://github.com/archlinux/archinstall/assets/83916775/797f8add-8dc2-4557-960c-d63ec03d1fe8)
1 kernel takes 233MB, with 2 kernel mkinitcpio -P fails because there is insufficient space.

**Comment by Torxed** [comment]
> 1 kernel takes 233MB, with 2 kernel mkinitcpio -P fails because there is insufficient space.

I understand, the kernel != EFI binaries.
In the future, archinstall will try to make the 2 kernels live in `/boot` and the `EFI` folder will live on a EFI partition.

The issue is if `EFI` and `/boot` is the same partition *(historically this is what archinstall have been using)* as then, in your example, the singular partition will be too small.

Now, we can have a `120MB` EFI even, and that will work.
As long as `/boot` *(different from EFI)* is large enough to contain the kernels.

tl;dr: We're fixing it ^^

**Comment by Cardiacman13** [comment]
Thx for your work :)

**Comment by Xdavius** [comment]
Hi toxed.
Are you saying the auto partitioning will create : 
1 EFI 130mo
1 Boot (sized to move than 512mo ?
1 root

Thank you for your answers.


I have also a lot of issues using custom sized partition made with cfdisk... or using the integrated tool (not very easy to use, lots of issues too)

Have you got a plan for this part ?

Regards.

Davius

**Comment by codefiles** [comment]
The ArchWiki now suggests an ESP size of 1 GiB.

https://wiki.archlinux.org/index.php?title=Installation_guide&diff=prev&oldid=799742
